specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/machineLearningServices.json

```diff
@@ -2670,55 +2683,82 @@
               "type": "array",
               "items": {
                 "$ref": "#/definitions/DiagnoseResult"
-              }
+              },
+              "x-ms-identifiers": [
+                "code"
```

Comment:
code is not unique, the message will be unique though.

Reply:
Thanks for pointing it out. I have change the identifiers to message in the new iteration.